pkg/dashboard/resource/project.go

```diff
@@ -160,6 +162,53 @@ func (pr *projectResource) Create(request *http.Request) (id string, attributes
 	return pr.createProject(request, projectInfo)
 }
 
+// Update a project
+func (pr *projectResource) Update(request *http.Request, id string) (restful.Attributes, error) {
+	ctx, cancelCtx := context.WithCancel(nucliocontext.NewDetached(request.Context()))
+	defer cancelCtx()
```

Comment:
Is it necessary to cancel the detached context?

Reply:
@yaelgen  just use `ctx := nucliocontext.NewDetached(request.Context())`